### PR TITLE
guest to guest chat eDiscovery

### PR DESCRIPTION
**Pull request by anwara-ms**
Today, guest to guest chat eDiscovery is not supported and the documentation of the feature is not correct.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.
--

File: Teams/eDiscovery-investigation.md [hunk 1]
```diff
@@ -38,11 +38,7 @@ All Teams 1:1 or group chats are journaled through to the respective users’ ma
 
 ## eDiscovery of guest-to-guest chats
 
-Without a mailbox, guest-to-guest chats (1xN chats in which there are no home tenant users) would not be indexed, and as a result, would not be included in eDiscovery. To facilitate eDiscovery for guest-to-guest chats, a cloud-based mailbox (or phantom mailbox) is created to store the 1xN data. After the Teams chat data is stored in the cloud-based mailbox, it is indexed for eDiscovery and compliance content search.
-
-The following illustration shows how eDiscovery works for guest-to-guest chats in which there isn’t a mailbox.
-
-![guest-to-guest-chats-with-no-mailbox](media/conduct-an-ediscovery-investigation-of-content-in-microsoft-teams-image2.png)
+Today, for a sceanrio where only guests are participating in 1:1 or 1:N chat, we dont support eDiscvoery of those chat messages.
```
Comment (JohanFreelancer9) [inline Teams/eDiscovery-investigation.md]:
```suggestion
Presently, for a scenario where only guests are participating in 1:1 or 1:N chat, we don't support eDiscvoery of those chat messages. 
```

Comment (LolaJacobsen) [reply Teams/eDiscovery-investigation.md]: Please fix the typo - "eDiscvoery"

Comment (JohanFreelancer9) [reply Teams/eDiscovery-investigation.md]: @LolaJacobsen  - Thanks for spotting that. Done.

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**